applications/callflow/src/cf_endpoint.erl

```diff
@@ -116,6 +118,14 @@ maybe_have_endpoint(JObj, EndpointId, AccountDb) ->
             has_endpoint(JObj, EndpointId, AccountDb, EndpointType)
     end.
 
+-spec maybe_map_endpoint_type(api_binary()) -> api_binary().
+maybe_map_endpoint_type('undefined') -> 'undefined';
+maybe_map_endpoint_type(EndpointType) ->
```

Comment:
Fine by me. Can you update the binary types to use the document functions (`kz_device:type()`, `kzd_user:type()`, etc...) where available?